.travis.yml

```diff
@@ -1,5 +1,5 @@
+sudo: true
 language: php
-
 php:
   - 7
```

Comment:
Ah we'd also need 5.6, 7.1 and 7.2 then :wink:

Reply:
Done. Obviously the build matrix grew but the test scripts are fast, so no big deal IMO